src/libtsduck/base/network/tsSRTSocket.cpp

```diff
@@ -1046,7 +1046,9 @@ int ts::SRTSocket::Guts::listenCallback(void* param, SRTSOCKET sock, int hsversi
     Guts* guts = reinterpret_cast<Guts*>(param);
     if (guts == nullptr || (guts->listener != SRT_INVALID_SOCK && guts->sock != SRT_INVALID_SOCK)) {
         // A connection is already established, revoke all others.
+#ifndef SRT_REJX_OVERLOAD
```

Comment:
why?

Reply:
Customizable rejection reason wasn't added until 1.4.2, added it under this flag instead of version for some reason https://github.com/Haivision/srt/commit/9b6b12443d93c631e1d35348e2d141cf41e79d91